tnt-transfer-obj/src/main/java/org/dcsa/tnt/transferobjects/EventSubscriptionWithSecretTO.java

```diff
@@ -0,0 +1,21 @@
+package org.dcsa.tnt.transferobjects;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+import javax.validation.constraints.NotBlank;
+
+@Data
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+public class EventSubscriptionWithSecretTO extends EventSubscriptionTO {
+  @NotBlank
+  private String secret;
```

Comment:
There are length limits on the secret, which have been lost in the R2DBC rewrite.

Reply:
Also, the secret was a base64 encoded value originally (`byte[]` having Jackson encode/decode it for us).  I do not see that happen in this PR either.

Reply:
Changed the type to byte[], but I did not see any limits in the r2dbc version so I guessed min 8 and max 1024 bytes.

Reply:
No, I think it is 32 and 64 bytes.  It is in he SignatureFunction or SignatureMethod class (look for the SHA256 variant - the PLAIN was just me testing)

Reply:
Changed now, min is 32 and max is 64